$OpenBSD$

Use newer graphics/qr-code-generator header from our port.

Fixed upstream, remove this patch upon the next update.

Index: Telegram/lib_qr/qr/qr_generate.cpp
--- Telegram/lib_qr/qr/qr_generate.cpp.orig
+++ Telegram/lib_qr/qr/qr_generate.cpp
@@ -8,7 +8,7 @@
 
 #include "base/assertion.h"
 
-#include "QrCode.hpp"
+#include "qrcodegen.hpp"
 #include <QtGui/QPainter>
 #include <string>
 
